js/modules/settings/radioConnectSetting.js

```diff
@@ -96,30 +97,22 @@ export class RadioConnectSetting extends Setting {
     _waitUntillSynced() {
         let connectingProgress;
         let syncingProgress;
+        let DeviceConnectionOverlay;
 
         connectingProgress = mdui.dialog({
             headline: "Connecting",
             body: `<mdui-button loading variant="text" style="color: rgb(var(--mdui-color-on-surface));" full-width>Connecting to device.</mdui-button>`,
         });
 
         this.device.events.addEventListener("onStatus", (status) => {
-            switch(status) {
-                case Meshtastic.Types.DeviceStatusEnum.DeviceConnected:
-                    if (connectingProgress.open == false) break;
-                    
+            if (status = Meshtastic.Types.DeviceStatusEnum.DeviceConnected) {
```

Comment:
Just double checking if this is meant to be `status = Meshtastic.Types.DeviceStatusEnum.DeviceConnected` instead of `status == Meshtastic.Types.DeviceStatusEnum.DeviceConnected` ?

Reply:
yeah, sorry been a hot minute since I've had to do js